app/helpers/budget_bar_helper.rb

```diff
@@ -5,7 +5,7 @@ def progress_bar_color(type)
     "bg-#{color}"
   end
 
-  def progress_bar_width(value, cash_ceiling = 1_000)
+  def progress_bar_width(value, cash_ceiling = Config.budget_bar_max)
```

Comment:
hm, I think you'll want to go up a level - take a look at `budget_bar_calculable.rb` which I think is what's invoking these methods. I think that's overruling the default arg. relatedly it's kinda unnecessary that we have a default arg here whatsoever, since this is being called from `BudgetBarCalculable#budget_bar_calculations`, so if we wanted to make these standard regular arguments instead that would probably be wise.

It does not surprise me in the least that I wrote this as some medium-spaghetti code 😬 😅